server/datastore/mysql/migrations/tables/20250828122736_CreateTableSoftwareTitleIconsTable.go

```diff
@@ -0,0 +1,31 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250828122736, Down_20250828122736)
+}
+
+func Up_20250828122736(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+		CREATE TABLE software_title_icons (
+			id INT AUTO_INCREMENT PRIMARY KEY,
+			team_id INT UNSIGNED NOT NULL,
+			software_title_id INT UNSIGNED NOT NULL,
+			storage_id VARCHAR(64) COLLATE utf8mb4_unicode_ci NOT NULL,
+			filename VARCHAR(255) NOT NULL,
+			created_at timestamp(6) DEFAULT CURRENT_TIMESTAMP(6),
+			UNIQUE KEY idx_unique_team_id_title_id_storage_id (team_id, software_title_id, storage_id),
+			FOREIGN KEY (software_title_id) REFERENCES software_titles(id),
+			FOREIGN KEY (team_id) REFERENCES teams(id)
```

Comment:
We should cascade updates/deletes on software title ID.

Also, we can't do a fkey on team ID because of `No team`. Might need to use the `global_or_team_id` convention used on software installers, for the same reasons.